### PR TITLE
Toggling 2 detection and updating announcement of 1 detection

### PR DESCRIPTION
**Pull request by unknownhad**
Toggling 2 detection and updating announcement of 1 detection

### CODE REVIEW

File: content/waf/change-log/2022-08-30.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+title: 2022-08-30
+type: table
+pcx_content_type: changelog
+weight: 894
```
Comment (pedrosousa) [inline content/waf/change-log/2022-08-30.md]:
```suggestion
weight: 893
```

File: content/waf/change-log/scheduled-changes.md [hunk 2]
```diff
@@ -23,41 +23,14 @@ layout: list
   </thead>
   <tbody>
     <tr>
-      <td>2022-08-22</td>
       <td>2022-08-29</td>
```
Comment (pedrosousa) [inline content/waf/change-log/scheduled-changes.md]: Should the announcement date be `2022-08-30` instead?

Comment (unknownhad) [reply content/waf/change-log/scheduled-changes.md]: Fixed. Thanks

File: content/waf/change-log/2022-08-30.md (hunk 1 above)
Comment (unknownhad) [reply content/waf/change-log/2022-08-30.md]: Fixed. Thanks

**Review by pedrosousa — APPROVED** [review]

**Review by vs-mg — APPROVED** [review]

**Pull request merged**